includes/class-windows-azure-replace-media.php

```diff
@@ -55,13 +64,21 @@ public function __construct() {
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_replace_media_script' ) );
 
 		// ajax event to replace media
-		add_action( 'wp_ajax_nopriv_azure-storage-media-replace', array(  $this, 'process_media_replacement' ) );
-		add_action( 'wp_ajax_azure-storage-media-replace', array(  $this, 'process_media_replacement' ) );
-
-		// Ajax event to set transient for replacement
-		add_action( 'wp_ajax_nopriv_azure-storage-media-replace-set-transient', array(  $this, 'set_media_replacement_transient' ) );
-		add_action( 'wp_ajax_azure-storage-media-replace-set-transient', array(  $this, 'set_media_replacement_transient' ) );
+		add_action( 'wp_ajax_nopriv_azure-storage-media-replace', array( $this, 'process_media_replacement' ) );
```

Comment:
I know this is not part of this PR but does this need to fire on the `nopriv` hook? Seems like replacing media is something that should only be triggered by a logged in user